rtmp-client/src/main/cpp/librtmp/rtmp.h

```diff
@@ -364,10 +395,10 @@ extern "C"
   int RTMPSockBuf_Send(RTMPSockBuf *sb, const char *buf, int len);
   int RTMPSockBuf_Close(RTMPSockBuf *sb);
 
-  int RTMP_SendCreateStream(RTMP *r);
-  int RTMP_SendSeek(RTMP *r, int dTime);
-  int RTMP_SendServerBW(RTMP *r);
-  int RTMP_SendClientBW(RTMP *r);
+  RTMPResult RTMP_SendCreateStream(RTMP *r);
+  RTMPResult RTMP_SendSeek(RTMP *r, int dTime);
+  RTMPResult RTMP_SendServerBW(RTMP *r);
+  RTMPResult RTMP_SendClientBW(RTMP *r);
```

Comment:
It would be helpful if RTMP_Read and RTMP_Write below could be documented with what they return on success and failure. And in particular what RTMP_Read does if the stream has ended.

Reply:
I'll add comments.